src/transformState.js

```diff
@@ -5,7 +5,24 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
```

Comment:
I think it will be better.
```suggestion
  for (const action of actions) {
    const {type, extraData, keysToRemove} = action;
    
    switch (type) {
```